app/cdap/components/SourceControlManagement/LocalPipelineListView/CommitModal.tsx

```diff
@@ -82,6 +84,9 @@ export const CommitModal = ({
       setPushError(res.message);
       setPushStatus(res.status);
       setPushLoadingMessage(null);
+      if (res.fileHash && typeof updateConnectedChipStatus === 'function') {
+        updateConnectedChipStatus(res.fileHash);
```

Comment:
It `updateConnectedChipStatus` too specific?

Reply:
changed